LayoutTests/imported/w3c/web-platform-tests/trusted-types/set-event-handlers-content-attributes.tentative-expected.txt

```diff
@@ -455,4 +455,10 @@ PASS AUDIO.setAttribute(onwaitingforkey, "unsafe_handler()") calls default polic
 PASS AUDIO.setAttributeNS(onwaitingforkey, "unsafe_handler()") calls default policy
 PASS VIDEO.setAttribute(onwaitingforkey, "unsafe_handler()") calls default policy
 PASS VIDEO.setAttributeNS(onwaitingforkey, "unsafe_handler()") calls default policy
+PASS DIV.setAttribute("onafterprint", "unsafe_handler()") calls default policy
+PASS DIV.setAttribute("onwaitingforkey", "unsafe_handler()") calls default policy
+FAIL DIV.setAttribute("onbegin", "unsafe_handler()") calls default policy assert_equals: expected (string) "Element onbegin" but got (undefined) undefined
+FAIL DIV.setAttribute("onreadystatechange", "unsafe_handler()") does not call default policy assert_equals: expected (undefined) undefined but got (string) "Element onreadystatechange"
+FAIL DIV.setAttribute("onopen", "unsafe_handler()") does not call default policy assert_equals: expected (undefined) undefined but got (string) "Element onopen"
```

Comment:
These two are expected to fail in WebKit due to the mechanism used for protecting event handlers. These failures do NOT represent as TT bypass.